projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_router.py

```diff
@@ -226,10 +226,10 @@ def __initialize_ingester(self, method) -> IngesterBase:
         else:
             # Initialize the pre- and post- processors.
             initialized_pre_processors = self.__get_initialized_processors(
-                "INGEST_PAYLOAD_PREPROCESS_SEQUENCE"
+                "VDK_INGEST_PAYLOAD_PREPROCESS_SEQUENCE"
```

Comment:
It should be without vdk_

Reply:
Yes, VDK_ prefix is now removed in supercollider-vdk, which is the correct way to fix this mismatch which lead to never loading pre-processors in our test case.